types/focus-within/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for focus-within 1.0
+// Project: https://github.com/jonathantneal/focus-within#readme
+// Definitions by: Damien Erambert <https://github.com/eramdam>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare global {
+interface FocusWithinOpts {
+    attr?: boolean;
+    className?: string;
+}
+
+type FocusWithin = (document: HTMLDocument, opts?: FocusWithinOpts) => void;
+
+const focusWithin: FocusWithin;
+}
+
+export default focusWithin;
```

Comment:
should be `export = focusWithin`. see https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default for more details